dpctl/tensor/_elementwise_funcs.py

```diff
@@ -261,7 +261,27 @@
 # FIXME: implement U25
 
 # B20: ==== NOT_EQUAL   (x1, x2)
-# FIXME: implement B20
+_not_equal_docstring_ = """
+not_equal(x1, x2, order='K')
```

Comment:
Add `out` keyword, and describe it in Args section. Look at @vtavana PR for sin/exp for my changes to `abs` docstring.